Backend/credentials/test_credentials.py

```diff
@@ -0,0 +1,47 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+from JobListings.models import Job, Employer
+from requests.auth import HTTPBasicAuth
+from django.contrib.auth.models import User
+
+from credentials.models import Candidate
+
+# Create your tests here.
+class TestCandidate(APITestCase):
+    base_url = reverse('candidate-list')
+
+    def setUp(self):
+        password = 'Password'
+        my_admin = User.objects.create_superuser('admin', 'admin@linkedout.com', password)
+        self.client.login(username=my_admin.username, password=password) # login authentication
+
+        Candidate.objects.create(first_name = "John", last_name = "Doe")
+        Candidate.objects.create(first_name = "Larry", last_name = "Page")
+
+    def test_get_a_candidate(self):
+        url = self.base_url + '1/'
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data['first_name'], 'John')
+        self.assertEqual(response.data['last_name'], 'Doe')
+
+    def test_get_all_candidates(self):
+        url = self.base_url
+        candidate_count = Candidate.objects.all().count()
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data), candidate_count)
```

Comment:
Would we be able to assert that the values inside the response match those found in the database, instead of asserting the length of the response?

Reply:
I think it could be possible, the response data is JSON I believe and fetching from the database can also give the results in a json format. Meaning that by making the JSON values into string and using assertEqual should be enough

Reply:
Yeah i could add that, but i just wanted to test that we get all the data back, since the individual fields are validated in test_get_a_job.

Reply:
Okay so the fields are already verified and you're just checking for the same number of fields